templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Confirm deletion</h1>
+  <p>Do you really want to delete the "<strong>{{ car.model }}</strong>" car?</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Delete" class="btn btn-danger">
+  </form>

```

Comment:
It would be great to add a `Cancel` button to all the forms.

Reply:
Thanks, good idea